src/invite.tsx

```diff
@@ -15,13 +15,15 @@ import { CreateWalletAccountContainer } from './authentication/create-wallet-acc
 
 export interface Properties {
   stage: RegistrationStage;
+  shouldRender: boolean;
 }
 
 export class Container extends React.Component<Properties> {
   static mapState(state: RootState): Partial<Properties> {
-    const { registration } = state;
+    const { registration, pageload } = state;
     return {
       stage: registration.stage,
+      shouldRender: pageload.loadPage,
```

Comment:
Just wanted to point out how awesome this subtle decision was. The state variable is named based on the context of the state and the component property is named based on it's impact. :rocket: